libgramtools/src/kmer_index/build.cpp

```diff
@@ -39,68 +39,45 @@ CacheElement get_initial_cache_element(const Base &base,
             SA_Interval{0, prg_info.fm_index.size() - 1}
     };
     SearchStates search_states = {search_state};
-    CacheElement initial_cache_element = {search_states};
+    CacheElement full_SA_interval = {search_states};
```

Comment:
I don't think you should capitalize SA in variable names. I suggest: `full_sa_interval`.